lib/active_record/connection_adapters/oracle_enhanced/context_index.rb

```diff
@@ -160,7 +160,7 @@ def create_datastore_procedure(table_name, procedure_name, column_names, options
                     "l_#{col} VARCHAR2(32767);\n"
                   end.join
                 end.join
-                } BEGIN
+              } BEGIN
```

Comment:
The following may make the pair of parentheses even clearer.

```diff
@@ -153,14 +153,14 @@ module ActiveRecord
                 (p_rowid IN          ROWID,
                 p_clob IN OUT NOCOPY CLOB) IS
                 -- add_context_index_parameters #{(column_names + select_queries).inspect}#{!options.empty? ? +', ' << options.inspect[1..-2] : ''}
-                #{
+              #{
                 selected_columns.map do |cols|
                   cols.map do |col|
                     raise ArgumentError, "Alias #{col} too large, should be 28 or less characters long" unless col.length <= 28
                     "l_#{col} VARCHAR2(32767);\n"
                   end.join
                 end.join
-                } BEGIN
+              } BEGIN
                 FOR r1 IN (
                   SELECT #{quoted_column_names.join(', ')}
                   FROM  #{quoted_table_name}
```

Reply:
Thanks for the review. Updated and pushed.